apps/mobile/src/hooks/create-wallet.ts

```diff
@@ -59,7 +59,7 @@ export function useCreateWallet() {
 
   async function navigateAndCreateWallet() {
     switch (securityLevelPreference) {
-      case 'undefined':
+      case 'not-selected':
```

Comment:
This switch should have type `securityLevelPreference: SecurityLevelPreference` right?